BranchMonsterFactory/ExternalLibs/Branch-SDK/BNCConfig.h

```diff
@@ -1,34 +0,0 @@
-//
```

Comment:
so we're removing the SDK in favor of the cocoapod? I'm certainly glad. Did you commit the change removing this `ExternalLibs` folder? git on my machine now says ExternalLibs is not staged for changing, I'll have to troubleshoot that separately.